src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -945,7 +945,7 @@ def migrate_to_multisite(self, realm_name: str, zonegroup_name: str, zone_name:
     def create_realm(self, realm_name: str, default: bool):
         rgw_realm_create_cmd = ['realm', 'create']
         cmd_create_realm_options = ['--rgw-realm', realm_name]
-        if default != 'false':
+        if default != False:
```

Comment:
if the value of default is bool, the f needs to be in small case
```suggestion
        if default != false:
```

Reply:
I think its with a capital 'F' in python, verified with loggers

Reply:
okay, I thought it was java script bool